src/shared/bus-locator.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+typedef struct BusLocator {
+        const char    *destination;
+        const char    *path;
+        const char    *interface;
+} BusLocator;
+
+extern const BusLocator *bus_home_mgr;
+extern const BusLocator *bus_import_mgr;
+extern const BusLocator *bus_locale;
+extern const BusLocator *bus_login_mgr;
+extern const BusLocator *bus_machine_mgr;
+extern const BusLocator *bus_network_mgr;
+extern const BusLocator *bus_portable_mgr;
+extern const BusLocator *bus_resolve_mgr;
+extern const BusLocator *bus_systemd_mgr;
+extern const BusLocator *bus_timedate;
```

Comment:
This should be `const BusLocator* const bus_...;`. Right now the pointers themselves are mutable.

Reply:
Duh, fixed.